modules/rest-http/src/main/java/org/apache/ignite/internal/processors/rest/protocols/http/jetty/GridJettyRestProtocol.java

```diff
@@ -123,10 +123,10 @@ public GridJettyRestProtocol(GridKernalContext ctx) {
         InetAddress locHost;
 
         try {
-            locHost = U.resolveLocalHost(ctx.config().getLocalHost());
+            locHost = U.resolveLocalHost(System.getProperty(IGNITE_JETTY_HOST, ctx.config().getLocalHost()));
         }
         catch (IOException e) {
-            throw new IgniteCheckedException("Failed to resolve local host to bind address: " + ctx.config().getLocalHost(), e);
+            throw new IgniteCheckedException("Failed to resolve local host to bind address", e);
```

Comment:
I think it is better to change to "Failed to resolve Jetty host to bind address: " + jettyHost ?

Reply:
or we can do this way
```
String jettyHost = System.getProperty(IGNITE_JETTY_HOST, ctx.config().getLocalHost());

try {
      System.setProperty(IGNITE_JETTY_HOST, U.resolveLocalHost(jettyHost).getHostAddress());
    }
    catch (IOException e) {
       throw new IgniteCheckedException("Failed to resolve host to bind address: " + jettyHost, e);
}
```

Reply:
@akuznetsov-gridgain Changed it to this way.